lib/common/zstd_internal.h

```diff
@@ -290,8 +290,10 @@ void ZSTD_wildcopy(void* dst, const void* src, ptrdiff_t length, ZSTD_overlap_e
         }
         while (op < oend);
 #else
-        COPY16(op, ip);
-        if (op >= oend) return;
+        memcpy(op, ip, 16);
```

Comment:
nit: Use `ZSTD_copy16()` to be consistent with `COPY16()`. This opts into explicit `vst1q_u8()` generation on aarch64.

Reply:
Thanks for pointing it out. The code is updated.